test/service/test_response.py

```diff
@@ -2035,24 +2042,50 @@ def test_summary_filter_none(self):
 
 class TestUnpopulatedIndexResponse(WebServiceTestCase):
 
+    @classmethod
+    def bundles(cls) -> List[BundleFQID]:
+        return []
+
     @classmethod
     def setUpClass(cls):
         super().setUpClass()
-        cls.index_service.create_indices(cls.catalog)
+        cls._setup_indices()
+        cls.maxDiff = None
 
     @classmethod
     def tearDownClass(cls):
-        cls.index_service.delete_indices(cls.catalog)
+        cls._teardown_indices()
         super().tearDownClass()
 
+    def facets(self) -> Sequence[str]:
+        return self.app_module.app.facets
+
+    entity_types = ['files', 'projects', 'samples', 'bundles']
```

Comment:
Don't hard-code this. There is a method in the DocumetService

Reply:
Borrowed from https://github.com/DataBiosphere/azul/blame/6c8c2f15c040cc1f8831a528d35b5b3795ea36ee/test/integration_test.py#L557